group/gt.go

```diff
@@ -45,15 +47,27 @@ func (z *GtElement) AddAssign(e Element) Element {
 	return z
 }
 
+func (z *GtElement) SubAssign(e Element) Element {
+	ee := e.(*GtElement).inner
+	z.inner.Div(&z.inner, &ee)
+	return z
+}
+
 func (z *GtElement) Add(a, b Element) Element {
 	z.Set(a)
 	z.AddAssign(b)
 	return z
 }
 
+func (z *GtElement) Sub(a, b Element) Element {
+	z.Set(a)
+	z.SubAssign(b)
+	return z
+}
+
 func (z *GtElement) MultiExp(basis []Element, scalars []fr.Element) (Element, error) {
 	// Maybe quite naive; but it works. Prob could use some Pippenger algorithm?
-	z.inner = bls12381.GT{}
+	z.inner.SetOne()
```

Comment:
The default `bls12381.GT{}` is the neutral element in the additive group. But we live in a multiplicative subgroup, so the correct neutral element is one, not zero.